habitat-lab/habitat/articulated_agents/articulated_agent_base.py

```diff
@@ -27,6 +27,7 @@ def __init__(
         fixed_based: bool = True,
         base_type="mobile",
         sim_obj=None,
+        maintain_link_order=False,
```

Comment:
I know there is no docstring for this, but can you add one for this one variable please?